src/Controller/PatternLibraryController.php

```diff
@@ -74,11 +77,12 @@ public function single($name) {
   public function overview() {
 
     $definitions = $this->patternsManager->getDefinitions();
-    foreach ($definitions as $name => $definition) {
-      $definitions[$name]['rendered']['#type'] = 'pattern_preview';
-      $definitions[$name]['rendered']['#id'] = $name;
-      $definitions[$name]['meta']['#theme'] = 'patterns_meta_information';
-      $definitions[$name]['meta']['#pattern'] = $definition;
+    foreach ($definitions as $definition) {
```

Comment:
Yes that's the second part if this PR, I'm gonna create an actual object for each pattern so we can re-organize the manager too.